app/volume/template.hbs

```diff
@@ -0,0 +1,132 @@
+<section class="header clearfix">
+  <h1 class="pull-left">
+    <a href="{{href-to 'volumes'}}">{{t 'volumesPage.title'}}</a>: {{model.name}}
+  </h1>
+  <div class="right-buttons">
+    {{badge-state model=model}}
+    {{action-menu model=model showPrimary=false classNames="ml-10 inline-block" size="sm"}}
+  </div>
+</section>
+
+{{#if model.description}}
+  {{banner-message color='bg-secondary mb-0 mt-10' message=model.description}}
+{{/if}}
+
+<section>
+  <div class="banner bg-info basics">
+    {{#if model.driver}}
+      <div class="inline-block">
+        <label class="acc-label p-0">{{t 'volumesPage.banner.driver'}}:</label>
+        {{#copy-inline clipboardText=model.driver}}
+          {{model.driver}}
+        {{/copy-inline}}
+      </div>
+    {{/if}}
+    {{#if model.sizeMb}}
+      <div class="inline-block">
+        <label class="acc-label p-0">{{t 'volumesPage.banner.size'}}:</label>
+        {{model.sizeMb}}
+      </div>
+    {{/if}}
+    {{#if model.stackId}}
+      <div class="inline-block">
+        <label class="acc-label p-0">{{t 'volumesPage.banner.stack'}}:</label>
+        <a href="{{href-to 'stack' stack.id}}">{{stack.displayName}}</a>
+      </div>
+    {{/if}}
+    {{#if model.perContainer}}
+      <div class="inline-block">
+        <label class="acc-label p-0">{{t 'volumesPage.banner.perCont'}}:</label>
+        {{model.perContainer}}
+      </div>
+    {{/if}}
+    {{#if driver.uri}}
+      <div class="inline-block">
+        <label class="acc-label p-0">{{t 'volumesPage.banner.uri'}}:</label>
+        {{#copy-inline clipboardText=model.uri}}
+          <div class="clip" style="vertical-align:bottom;max-width:200px;">
+            {{model.uri}}
+          </div>
+        {{/copy-inline}}
+      </div>
+    {{/if}}
+    {{#if model.hostId}}
+      <div class="inline-block">
+        <label class="acc-label p-0">{{t 'volumesPage.banner.host'}}:</label>
+        <a href="{{href-to 'host' host.id}}">{{host.displayName}}</a>
+      </div>
+    {{/if}}
+  </div>
+</section>
+
+<section>
+  {{#accordion-list as |al expandFn| }}
+
+      {{#accordion-list-item
+           title=(t 'volumesPage.mounts.label')
+           detail=(t 'volumesPage.mounts.detail')
+           status=(t 'volumesPage.mounts.status' count=(or model.mounts.length 0))
+           statusClass=(if model.mounts.length 'bg-success' 'text-muted')
+           expandOnInit=true
+           expandAll=al.expandAll
+           expand=(action expandFn)
+           componentName='sortable-table'
+           as | parent |
+      }}
+        {{#component parent.intent
```

Comment:
Prevents the child component being rendered until the parent is expanded. I could remove the intent from the mounts list becuase that has `expandOnInit=true` so there are no render savings there.